src/Cart/CartProduct.js

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable max-len */
```

Comment:
Remove this comment, if lines are over the max length just split them into 2 lines